docs/release/release-1.5.1.rst

```diff
@@ -0,0 +1,9 @@
+=============
+Release 1.5.1
+=============
+
+This release works for Presto versions 0.116 through at least 0.152.1
+
+Bug Fixes
+---------
+* Pin the Docker images used for the product tests to a specific version
```

Comment:
I think release notes should be user facing changes.  Though I guess we need some explanation for a new release.